rasa/nlu/featurizers/sparse_featurizer/lexical_syntactic_featurizer.py

```diff
@@ -70,15 +74,35 @@ def required_components(cls) -> List[Type[Component]]:
         "digit": lambda token: token.text.isdigit(),
     }
 
+    _function_dict_lower: Dict[Text, Callable[[Token], Union[bool, Text, None]]] = {
+        "prefix5": lambda token: token.text[:5].lower(),
+        "prefix2": lambda token: token.text[:2].lower(),
+        "suffix5": lambda token: token.text[-5:].lower(),
+        "suffix3": lambda token: token.text[-3:].lower(),
+        "suffix2": lambda token: token.text[-2:].lower(),
+        "suffix1": lambda token: token.text[-1:].lower(),
+    }
+
     def __init__(
         self,
         component_config: Dict[Text, Any],
         feature_to_idx_dict: Optional[Dict[Text, Any]] = None,
     ):
+        """Create a LexicalSyntacticFeaturizer.
```

Comment:
```suggestion
        """Creates a LexicalSyntacticFeaturizer.
```

Reply:
Won't pass flake8. Your linter is set to "very very picky"

Reply:
@JEM-Mosig mind to resolve?

Reply:
I can't seem to reproduce this... `make lint` runs fine for these changes. Can you let me know what commands you ran?

Reply:
I used the flake8 commands locally that are part of `make lint` and it complained about the mood of the first line, should be imparative.
Commit https://github.com/RasaHQ/rasa/pull/10482/commits/9923e9f0fb3f4d8be6ccd6f451dd3ccc4ea7564d just had a green light on **passing all tests**. I will re-check by committing this into the branch. Let's see if my local flake8 was even pickier...